tools/genjava.js

```diff
@@ -210,9 +210,12 @@ function processPOM(pom) {
 foam.poms.forEach(processPOM);
 console.log(`GENJAVA: Found ${found} java files.`);
 
-//console.log(X.javaFiles);
-fs_.writeFileSync('./target/javaFiles', X.javaFiles.join('\n') + '\n');
-fs_.writeFileSync('journalFiles',       X.journalFiles.join('\n') + '\n');
+// Only overwrite javaFiles when genjava:true
+if ( flags.genjava )
+  fs_.writeFileSync('./target/javaFiles', X.javaFiles.join('\n') + '\n');
+
+// REVIEW: outputJournals() should already generate all journal.0 files, writing to journalFiles is not needed
+// fs_.writeFileSync('journalFiles',       X.journalFiles.join('\n') + '\n');
```

Comment:
I don't see that outputJournals() outputs these, but this was just for backward compatibility and isn't actually needed.

Reply:
The last line of addJournal() writes the filename before its content to journal.0 file. Couldn't find the place that use the `javaFiles` file, maybe safe to stop writing out the file but I can revert if we want to keep the behavior, @kgrgreer ?